src/example/rendering_particle.cpp

```diff
@@ -12,28 +13,34 @@
 #include <crepe/api/Sprite.h>
 #include <crepe/api/Texture.h>
 #include <crepe/api/Transform.h>
-#include <crepe/api/Vector2.h>
 #include <crepe/system/RenderSystem.h>
+#include <crepe/types.h>
 
 #include <chrono>
-#include <iostream>
 #include <memory>
 
 using namespace crepe;
 using namespace std;
 
 int main(int argc, char * argv[]) {
 	ComponentManager mgr;
-	GameObject game_object = mgr.new_object("", "", vec2{0, 0}, 0, 0.1);
+	GameObject game_object = mgr.new_object("", "", vec2{0, 0}, 0, 1);
 	RenderSystem sys{mgr};
 	ParticleSystem psys{mgr};
+	AnimatorSystem asys{mgr};
 
 	Color color(255, 255, 255, 255);
 
 	Sprite & test_sprite = game_object.add_component<Sprite>(
-		make_shared<Texture>("asset/texture/img.png"), color, FlipSettings{false, false});
+		make_shared<Texture>("asset/spritesheet/spritesheet_test.png"), color,
```

Comment:
Ik ben nu zelf ook aan het testen met een sprite zonder animator. Ik gebruik daarvoor de test_ap43.png. Deze sprite is 259 bij 195 px groot. Ik heb dit alsvolgt ingesteld:
![image](https://github.com/user-attachments/assets/5b285462-584f-439e-a2b4-67b4f537c557)
Daarnaast heb ik de Camera instellingen aangepast:
![image](https://github.com/user-attachments/assets/994608d5-8249-42ef-9a9a-7d3430d4f545)
Als resultaat krijg ik:
![image](https://github.com/user-attachments/assets/0b7cef20-68cb-47d3-8c6c-5004b52b5fb4)
Dit is niet wat ik verwacht. Ik zou gewoon de 4:3 image verwachten.

PS Dit is nadat ik een fix heb doorgevoerd zodat aspect_ratio niet wordt afgerond naar int (in Sprite.cpp).